boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp_iron.dts

```diff
@@ -14,15 +14,33 @@
  * the application core.
  */
 
+/ {
+	chosen {
+		zephyr,code-partition = &slot0_partition;
+		zephyr,uart-mcumgr = &uart136;
+	};
+};
+
 &mram1x {
 	cpuapp_rx_partitions: cpuapp-rx-partitions {
 		compatible = "nordic,owned-partitions", "fixed-partitions";
 		nordic,access = <NRF_OWNER_ID_APPLICATION NRF_PERM_RXS>;
 		#address-cells = <1>;
 		#size-cells = <1>;
 
-		cpuapp_slot0_partition: partition@2c000 {
-			reg = <0x2c000 DT_SIZE_K(480)>;
+		boot_partition: partition@2c000 {
+			label = "mcuboot";
+			reg = <0x2c000 DT_SIZE_K(64)>;
+		};
+
+		slot0_partition: partition@3c000 {
+			label = "image-0";
+			reg = <0x3c000 DT_SIZE_K(400)>;
+		};
+
+		slot1_partition: partition@6E000 {
+			label = "image-1";
+			reg = <0x6E000 DT_SIZE_K(400)>;
```

Comment:
I guess the size of these partitions was supposed to be 200 kB, not 400.

Reply:
Fixed in the new PR